tools/purge_archived_storages.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+require File.expand_path('../config/environment', __dir__)
+require "trollop"
+
+opts = Trollop.options do
```

Comment:
Not sure how important this is but we should probably be using `Optimist` here instead of `Trollop`

Reply:
Actually using trollop was the purpose here since hammer branch doesn't have optimist. https://github.com/ManageIQ/manageiq/pull/18902#issuecomment-505458975